solidity/compile.ts

```diff
@@ -17,7 +17,12 @@ const compileAugurConstantProductMarket = async () => {
 		sources: {
 			'AugurConstantProductMarket.sol': { content: await fs.readFile('contracts/AugurConstantProductMarket.sol', 'utf8') },
 			'AugurMock.sol': { content: await fs.readFile('contracts/AugurMock.sol', 'utf8') },
+			'Context.sol': { content: await fs.readFile('contracts/Context.sol', 'utf8') },
 			'DaiMock.sol': { content: await fs.readFile('contracts/DaiMock.sol', 'utf8') },
+			'draft-IERC6093.sol': { content: await fs.readFile('contracts/draft-IERC6093.sol', 'utf8') },
```

Comment:
What is `draft-IERC6093.sol` and why do we want/need it?

Reply:
ERC20 contract imports it:
import {IERC20Errors} from "./draft-IERC6093.sol";